app/models/manageiq/providers/azure/refresh_helper_methods.rb

```diff
@@ -84,6 +82,14 @@ def gather_data_for_this_region(arm_service, method_name = 'list_all')
     end
   end
 
+  def in_this_region(resources = nil)
```

Comment:
Something about this reads funny to me - perhaps something like `filter_my_region` or `filtered_by_region`? I'm not sure